src/Loader/NativeLoader.php

```diff
@@ -626,7 +629,7 @@ protected function createPropertyAccessor(): PropertyAccessorInterface
 
     protected function createCaller(): CallerInterface
     {
-        return new SimpleCaller($this->getCallProcessor(), $this->getValueResolver());
+        return new SimpleCaller($this->getCallProcessor(), $this->getValueResolver(), new NamedArgumentsResolver());
```

Comment:
I would add a `createNamedArgumentResolver()` method and add `@method NamedArgumentsResolver get NamedArgumentsResolver()`